src/Curation/Curation.router.ts

```diff
@@ -408,12 +408,12 @@ export class CurationRouter extends Router {
 
   private validateAccessToCuration = async (
     service: CurationService<any>,
-    id: string,
-    ethAddress: string
+    ethAddress: string,
```

Comment:
the function was being called with the `ethAddress` as the 2nd parameter. It was working correctly but only because they were being send in the wrong order to the `hasAccess` down below.

Reply:
Great catch!